src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -413,15 +412,12 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
       if (!hasReadyQuantities) {
         if (!noReadySeedlings) {
           setNoReadySeedlings(true);
-          snackbar.toastError(
-            strings.NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_DESCRIPTION,
-            strings.NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_TITLE
-          );
+          updateField('purpose', NurseryWithdrawalPurposes.NURSERY_TRANSFER);
+          setIsNurseryTransfer(true);
```

Comment:
i think we are missing initialization of from nursery... can you maybe extract out the 'onChangePurpose' and reuse that function?